app/src/main/java/com/example/aimovies/presentation/home/HomeScreen.kt

```diff
@@ -171,7 +146,7 @@ fun HomeScreenUi(
 @Preview(showBackground = true)
 fun HomeScreenPreview() {
     AIMoviesTheme {
-        HomeScreenUi(HomeUiModel(movies)) {
+        HomeScreenUi(HomeUiModel(emptyList())) {
```

Comment:
You can use HomeUiModel() without emptyList() as it has a default value of emptyList().